src/Error/BaseErrorHandler.php

```diff
@@ -270,6 +270,22 @@ protected function _logError($level, $data)
                 'start' => 1,
                 'format' => 'log'
             ]);
+			
+			$request = Router::getRequest();
+			if ($request) {
+				$message .= "\nRequest URL: " . $request->here();
+
+				$referer = $request->env('HTTP_REFERER');
+				if ($referer) {
+					$message .= "\nReferer URL: " . $referer;
+				}
+			
+				$clientIp = $request->clientIp();
+				if	($clientIp && $clientIp !== '::1') {
+					$message .= "\nClient IP: " . $clientIp;
+				}
+			}
```

Comment:
Perhaps we should have a helper method that adds request context to a log line instead of duplicating the code.